pyigm/cgm/cgmsurvey.py

```diff
@@ -0,0 +1,90 @@
+""" Classes for CGM Surveys
+"""
+import pdb
+
+from pyigm.utils import lst_to_array
+
+class CGMAbsSurvey(object):
+    """A CGM Survey class in absorption
+
+    Attributes
+    ----------
+    survey : str, optional
+      Survey name
+    ref : str, optional
+      Reference(s)
+    """
+    # Initialize with a .dat file
+    def __init__(self, survey='', ref=''):
+
+        # Name of survey
+        self.survey = survey
+        self.ref = ref
```

Comment:
Could the survey be a list of journal, vol, pg?  Then, reference info could be populated upon instantiation or at some other time.